src/entities/errors.ts

```diff
@@ -112,6 +112,7 @@ export class ErrorCodeUtils {
   ): ErrorCode {
     switch (backendErrorCode) {
       case BackendErrorCode.BackendStoreProblem:
+      case BackendErrorCode.BackendPaymentGatewayGenericError:
```

Comment:
This is to try to assign this error code to a know error instead of `Unknown`. Seemed like `StoreProblemError` fit the best...